### DISCUSSION TITLE
survival analysis with 300 lipids

### DISCUSSION THREAD

**Discussion by MoMaz123** [kogalur/randomForestSRC#249] [Q&A] [answered]
Hi, 
I want to run survival analysis with 300 lipids to find out the VIP variables, should I just use normal survival code? 
MMODEL <- rfsrc(Surv(time, status) ~ ., data = MYDTATA, 
               ntree = 100, block.size = 1)

Many thanks

**Comment by ishwaran** [comment, marked as answer]
For anti-VIMP (default)
`MMODEL <- rfsrc(Surv(time, status) ~ ., data = MYDTATA, importance = TRUE)`

For permute VIMP
`MMODEL <- rfsrc(Surv(time, status) ~ ., data = MYDTATA, importance = "permute")`


See also: https://www.randomforestsrc.org/reference/vimp.rfsrc.html and https://www.randomforestsrc.org/articles/vimp.html

**Comment by MoMaz123** [comment]
Many thanks,  what is anti-vimp please,

**Comment by ishwaran** [comment]
See the help file for `rfsrc` and read Section 6 on VIMP.  Most of your questions are already explained in the help file, with further references for reading, which you should first consult before posting here.  Thanks.

https://www.randomforestsrc.org/reference/rfsrc.html

**Reply by MoMaz123** [reply]
Many thanks,
